.github/workflows/regression_tests_cpu.yml

```diff
@@ -1,9 +1,9 @@
 name: Run Regression Tests on CPU
 
-on:
-  # runs every Sunday  at 11:15am
-  schedule:
-    - cron:  '15 11 * * 7'
+on: push
+  # runs every day  at 10:15am
```

Comment:
can we revert comments to all workflow files?

Reply:
Yes, I will revert it to on schedule when I commit.